### DISCUSSION TITLE
Protocol question

### DISCUSSION THREAD

**Discussion by MaricoList** [TheMicDiet/chihiros-led-control#55] [General]
Hi,
I have a question about settings parameters after power outage. First of all thanks for rev-engering the communication protocol it helps a lot get rid of Chichiros app and wifi Hub which  is a madness.
Does settings previously set with comnand id 165 mode 5 and auto mode CMD id 90 mode 5  are stored in non volatile memory in the LED' controller or must be re-set (with current time) on each power on cycle? In other words does controller after power on require only update the current time (CMD id 90 mode 9) and they will back on previously saved schedule or it need send all commands to set schedule,ramp, auto mode and time?

**Comment by TheMicDiet** [comment]
Hi @MaricoList, I would assume that the settings are restored after a power outage but I am not 100 % sure.

**Reply by MaricoList** [reply]
My second question is, does WRGBs and ARGB series should always respond  TX notify after each command sent? Currently I'm testing a custom controller (based on ESP32) with ARGB lamp and it reacts on command but it does not send any respond data on TX service. Second strange thing is ARGB ignore crc byte in set manual command.   It executes command regardless crc value or it may be absent.
